snap/local/hooks/cmd/configure/configure.go

```diff
@@ -250,8 +391,33 @@ func main() {
 
 	}
 
-	if err = handleAllServices(); err != nil {
+	install, err := cli.Config("install")
+	if err != nil {
+		fmt.Println(fmt.Sprintf("edgexfoundry:configure: reading 'install': %v", err))
+		os.Exit(1)
+	}
+
+	if install == "true" {
+		installMode = true
+	}
+
+	if err, startServices = handleAllServices(installMode); err != nil {
```

Comment:
a comment that handleAllServices will dynamically stop/start services if installMode is false would be nice

Reply:
otherwise it seems that handleAllServices just returns the set of services which should be started, but really it only returns the list of services that should be started when we are in install mode